platform/linux-generic/pktio/socket_xdp.c

```diff
@@ -289,6 +292,13 @@ static int sock_xdp_stats(pktio_entry_t *pktio_entry, odp_pktio_stats_t *stats)
 		stats->in_errors += qi_stats.errors;
 		stats->out_octets += qo_stats.octets;
 		stats->out_packets += qo_stats.packets;
+
+		if (!getsockopt(xsk_socket__fd(sock->xsk), SOL_XDP, XDP_STATISTICS, &xdp_stats,
```

Comment:
After this modification `odp_pktio_stats_reset()` may not reset all counters to zero.